README.md

```diff
@@ -72,3 +73,37 @@ all_models = app.list_models()
 all_llm_community_models = App().list_models(filter_by={"query": "LLM",
                                                         "model_type_id": "text-to-text"}, only_in_app=False)
 ```
+
+## Interacting with Workflows
+
+### Workflow Predict
+```python
+# Note: CLARIFAI_PAT must be set as env variable.
+from clarifai.client.workflow import Workflow
+
+# Workflow Predict
+workflow = Workflow(user_id="user_id", app_id="app_id", workflow_id="model_id")
+workflow_prediction = workflow.predict_by_url(url="url", input_type="image") # Supports image, text, audio, video
+
+# Customizing Workflow Inference Output
+workflow = Workflow(user_id="user_id", app_id="app_id", workflow_id="workflow_id",
+                  output_config={"min_value": 0.98}) # Return predictions having prediction confidence > 0.98
```

Comment:
Question out of curiosity. How does this min_value work if I have multiple models in a workflow? Do all the models in my workflow need to satisfy this constraint I define here?

Reply:
According to [endpoint](https://github.com/Clarifai/clarifai/blob/4ec4141c6c5cecd371ea77672e2a6bcce50f4f31/proto/clarifai/api/service.proto#L9152) comments I guess this output_config applies to all models in workflow.